pkgs/tools/misc/poweralertd/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchurl, meson, ninja, pkg-config, scdoc, systemd }:
+
+stdenv.mkDerivation rec {
+  pname = "poweralertd";
+  version = "0.1.0";
+
+  src = fetchurl {
+    url = "https://git.sr.ht/~kennylevinsen/poweralertd/archive/${version}.tar.gz";
+    sha256 = "11b0lgrp7kyysvc8gh8w37ydig74j2bg0y885ir73w3pym6sy65a";
+  };
+
+  patchPhase = ''
+    substituteInPlace meson.build --replace "systemd.get_pkgconfig_variable('systemduserunitdir')" "'${placeholder "out"}/lib/systemd/user'"
+  '';
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+    scdoc
+    systemd
+  ];
+
+  meta = with stdenv.lib; {
+    description = "UPower-powered power alerter";
+    homepage = "https://git.sr.ht/~kennylevinsen/poweralertd";
+    maintainers = with maintainers; [ thibautmarty ];
```

Comment:
Please add yourself as a maintainer in separate commit with the message ``maintainers: add thibautmarty``.